dagmc_stats.py

```diff
@@ -3,23 +3,21 @@
 sys.path.append('/opt/tljh/user/lib/moab/lib/python3.6/site-packages/pymoab-5.1.0-py3.6-linux-x86_64.egg')
```

Comment:
this should be change at some point in the future....

Reply:
This is a consequence of the current development environment....